b2sdk/file_version.py

```diff
@@ -62,8 +62,11 @@ def __init__(
         action,
         content_md5=None,
         server_side_encryption: Optional[EncryptionSetting] = None,  # TODO: make it mandatory in v2
-        legal_hold: Optional[bool] = None,
-        file_retention: Optional[FileRetentionSetting] = None,
+        file_retention: Optional[
+            FileRetentionSetting
+        ] = None,  # TODO: in v2 change the default value to NO_RETENTION_FILE_SETTING
+        legal_hold: Optional[LegalHold
```

Comment:
also - Paweł insisted on different order of `legal_hold` and `file_retention`

Reply:
I'm aware of Paweł's intention. In commit `Consistently place legal_hold after file_retention in all methods` `982a3907` many functions have been refactored to followe the order of `file_retention, legal_hold`. Thus I changed it the same way here